js/index.js

```diff
@@ -40,3 +40,78 @@ for (let i = 0; i < skills.length; i++) {
 
 // Save and refresh your browser
 // You should see your list of skills beneath the "Skills" heading
+
+// Handle Message Form Submit
+// Using "DOM Selection", select the "leave_message" form by name attribute and store it in a variable named messageForm
+const messageForm = document.querySelector('form[name="leave_message"]');
```

Comment:
This works, but feel free to just use id, since it's shorter: There's little reason to use name on non-input elements.